packages/forklift-console-plugin/src/modules/Plans/views/details/components/SettingsSection/components/NetworkNameTemplate/utils/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const networkNameTemplateAllowedVariables = [
+  '- .NetworkName: If target network is multus, name of the Multus network attachment definition, empty otherwise.',
+  '- .NetworkNamespace: If target network is multus, namespace where the network attachment definition is located.',
+  '- .NetworkType: type of the network ("Multus" or "Pod")',
+  '- .NetworkIndex: sequential index of the network interface (0-based)',
+];
```

Comment:
trans?

Reply:
adding to variables, the example patterns makes no sense to add those since it's should be only in English the yaml